Car

mjwarrior opened the pull request:
Test

marcjosephstreeter committed:
updated the time for the kids 

upped time to 2 hours!

marcjosephstreeter committed:
Merge pull request #1 from marcjosephstreeter/updated-game-time

updated the time for the kids

marcjosephstreeter committed:
Added new ways to earn time

marcjosephstreeter committed:
Update README.md

marcjosephstreeter committed:
fixed it to your standards lol!

Commit:
new ways to earn time

mjwarrior committed:
test

mjwarrior committed:
revert changes

mjwarrior committed:
Merge branch 'master' into car

mjwarrior committed:
update test

mjwarrior committed:
triggered

mjwarrior committed:
Here is my list of rules for the car approve or disapprove of them.

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,6 @@
+	1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+	2.	Passengers are limited to Katrina Wills and family. Any other passengers require permission.
+	3.	The car must be kept clean at all times.
+	4.	For any personal outings not related to schooling or church, fuel usage must be calculated in advance and paid for by you.
```
other things:


7 - must have and keep rules surrounding schedule
8 - must put in details regarding revocation
9 - must put in details regarding gas / insurance / maintenance expenditure

mjwarrior replied:
Ownership - delineate who owns what

mjwarrior committed:
I've edited it to your specifciations however I  added my twist.

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,31 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
```
add (permission is given with a pull request to a file "REQUESTLOG.MARCJOSEPH.md")

- `<date> <request>`

marcstreeter replied:
update the "restaurants" to be "restaurants within 20 mile radius of home"

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,31 @@
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+2.	Passengers are limited to Katrina Wills and family. Any other passengers require permission.
```
make copy "Only allowed passengers are Mom and/or Dad.  No other passengers (including siblings) are currently allowed. Exceptions require a pull request for each occurrence."

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,31 @@
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+2.	Passengers are limited to Katrina Wills and family. Any other passengers require permission.
+3.	The car must be kept clean at all times.
```
- no food, no trash (vacuuming isn't so important, and neither is car wash)

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,31 @@
+## Gas Maintenance and Fueling
+maintenance considerations
+
+1.	Gas will be provided only for to and from school; any personal outings are under your responsibility for fuel costs. For personal outings not related to schooling or church, fuel usage must be calculated in advance and paid for by you. Maintenance of the vehicle will be done by you unless someone else damages it or puts it in a position where it could be damaged. Insurance for the car is your  ...
```
Parents will put $60 gas each week, and car should never be brought home with less than 1/4 of a tank. 

Parents will put $100 maintenance every two months, and car may not be driven unless car meets maintenance standards (create a section that delineates car maintenance standards). Parents will pay for car insurance depending on perks (write section about home perks)

mjwarrior replied:
Please elaborate on perks

marcstreeter replied:
update: make it "up to $50 bi-weekly for gas and up to $50 bi-monthly maintenance (to be reviewed as needed)"

marcstreeter replied:
and put that under "Perks" section

marcstreeter replied:
and I'm not going to require you calculate fuel in advance.  We can just talk about it and make a pull request for the specific outing.

mjwarrior committed:
new set of rules

mjwarrior committed:
Made as requested made some tweaks to some rules you may like however you need to elaborate on perks define perks and I can put it on the readme

mjwarrior committed:
permanent revocation edited off

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,32 @@
+## Revocation requirements 
+1.	Failure to follow the rules may result in revocation of vehicle usage privileges.
+2.	Vehicle usage privileges may be revoked based on a three-strike system:
+  - The first strike will result in the removal of personal outings, such as visiting friends.
```
vehicle privileges will be revoked based on the severity or repeat offenses. For repeat offenses we will use a 3 strike system (first strike removes car access for 2 days, second strike 2 weeks, third strike 1 month, 4th strike total revocation).

marcstreeter replied:
we can reset strikes every year. Strikes must be listed in the "STRIKELOG.MARCJOSEPH.md" to count

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,32 @@
+
+1. Gas will be provided only for trips to and from school,church, work or job interviews; any personal outings are your financial responsibility. For personal outings not related to schooling or the above, fuel usage must be calculated in advance and paid for by you. Parents will contribute $100 for maintenance every two months and $60 for fuel weekly, the car may not be driven unless it meets ma ...
+### Scheduling 	
+blurb
```
I'm sure you mean to update this to not be "blurb"

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,32 @@
+### Scheduling 	
+blurb
+
+1.	Your schedule must be edited and tuned for the day before using the car.
```
and you must leave to your destination at least 10 minutes prior to the time "maps" says it will take to get there (ie: maps says that it will take 30 minutes to get there, you must leave 40 minutes prior to your appointment)

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,32 @@
+  - The third strike serves as a final warning.
+    Any additional strikes will lead to permanent revocation of vehicle usage. Strikes will be issued if any rule is violated, and they reset monthly.
+## Ownership 
+1.	Any items in the car that belong to you, and are used, will remain your property. If the vehicle is revoked, any and all items inside the car that are yours must be returned or compensated for accordingly.
```
add a line that states - "Ultimate ownership of the car is currently upon the parents, if parents need the vehicle they will always get priority"

marcstreeter replied:
have we stated anywhere the funds that will be made available each month for gas and upkeep?

mjwarrior committed:
Made the requested revisions.

mjwarrior committed:
made the change as requested

mjwarrior committed:
as requested

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,35 @@
+## Gas Maintenance and Fueling
+maintenance considerations
+
+1. Gas will be provided only for trips to and from school,church, work or job interviews; any personal outings are your financial responsibility. For personal outings not related to schooling or the above, fuel usage must be calculated in advance and paid for by you. Parents will contribute $100 for maintenance every two months and $60 for fuel weekly, the car may not be driven unless it meets ma ...
```
refer to an outdated comment for the updated copy

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,35 @@
+  - Third strike after this complete revocation 
+    Any additional strikes will lead to a revocation up to 6 months of vehicle usage. Strikes will be issued if any rule is violated, and they reset every 6 months .
+## Ownership
+1.  Ultimate ownership of the vehicle is to the parents if parents need the vehicle priority is given to them over you. 
```
you don't have to put "over you" -- its over anyone (but that can be implied).

marcstreeter commented on README.MARCJOSEPH.md:
```diff
@@ -0,0 +1,35 @@
+## Ownership
+1.  Ultimate ownership of the vehicle is to the parents if parents need the vehicle priority is given to them over you. 
+2.	Any items in the car that belong to you, and are used, will remain your property. If the vehicle is revoked, any and all items inside the car that are yours must be returned or compensated for accordingly.
+
```
lets make it clearer, that you have to remove your belongings and leave the car in a clean state.

marcstreeter replied:
and there is no guarantee of any compensation. The car is a transport and not a guaranteed locker. To protect yourself, keep it empty.

marcstreeter commented on REQUESTLOG.MARCJOSEPH.md:
put an example request log entry for future use

mjwarrior committed:
Chnages for pull requests and pull requests made review changes

marcstreeter commented on REQUESTLOG.MARCJOSEPH.md:
```diff
@@ -0,0 +1,15 @@
+- for denial: "10/23/2017 5:454 pm Regina to Margerita's Pizzeria at 10:45 pm 10/2024/2017| request [DENIED] reason: too late over curfew & short notice" 
+
+## Disputes
+1. Disputes can be over call, over text, or in person
```
but final results are recorded in the PR - helps everyone including the parent learn

marcstreeter commented on REQUESTLOG.MARCJOSEPH.md:
```diff
@@ -0,0 +1,15 @@
+# How to pull request
+1. (Date & Time of Request) i.e "10/23/2017 5:45 pm" (Passenger & Destination in Question) "Regina to Margerita's Pizzeria" add date and and estimated time of pickup "at/for 10:45 pm 10/24/2017" add the request bracket and put "| requested []" the parent then approves or denies the request if denial is given a reason has to be given and can be up for a dispute a reason can look like this "reason: ...
```
Kind of confusing -- lets go with this, more verbose, but queryable and programmable
```
[
  {
    date: string, // use YYYYMMDD represents day requested, not day request made
    persons: [
      {
        name: string,
        age: int,
        relationship: string, // friend, sibling
      },
    ],
    events: [
      {
        name: string, // pick up, drop off, accompany
        location: string, // address
        urls: [
          {
            type: string, // invite, google map url
            text: string, // https://...
          },
        ],
        timestamp: string, // use RFC3339
        duration: string,  // 3h, 15m, 0
      },
    ],
  }
]
```
that would mean we change the file type to `json`

mjwarrior committed:
Update README.MARCJOSEPH.md

Made requested changes to Gas and Maintenance, Scheduling, Revocation, and Ownership.
